docs/sphinx/specification/cudaq/operations.rst

```diff
@@ -99,4 +99,8 @@ qubit state is a :code:`|0>` (positive polarity) or :code:`|1>` (negative polari
 By default all control qubits are assumed to convey positive polarity. 
 The syntax for negating the polarity is the not operator preceeding the
 control qubit (e.g., :code:`x<cudaq::ctrl>(!q, r)`, 
-for :code:`cudaq::qubits` :code:`q` and :code:`r`).
\ No newline at end of file
+for :code:`cudaq::qubits` :code:`q` and :code:`r`). Negating the polarity of
+control qubits is supported in :code:`swap` and the gates in sets :code:`NAME`
```

Comment:
What does NAME and ROTATION_NAME refer to? I'll check once the docs build completes, but I think that will show up in the docs in that form.

Reply:
See lines 88 and 89.